data/defscript/popups.kvs

```diff
@@ -18,17 +18,17 @@ defpopup(highlight)
 		}
 	}
 
-	item($tr("Stop Highlighting (Local)","defscript"),41) (%:beingHlTmp && !%:beingHl)
+	item($tr("Stop highlighting (Local)","defscript"),41) (%:beingHlTmp && !%:beingHl)
```

Comment:
Heh.. these capitalization issues always bothered me.
Several UI guides (including the KDE one) suggest to use Title Case for menus, buttons and other simple controls.
This is what I have done in most places. However I agree that it's strongly debatable.
If it wasn't for the guide I'd use normal capitalization (the one you suggest to change to).
Whatever we choose to do, though, must be done for all the menus. That is, we need
a standard to follow everywhere...